rpc-client/src/subcommands/blockchain_subcommands.rs

```diff
@@ -40,8 +40,8 @@ pub enum BlockchainCommand {
         block_number: Option<u32>,
 
         /// Include transactions
```

Comment:
Should also update this rustdoc comment to clarify the whole body is included, which includes other things in macro blocks, as @onmax noticed.